### PR TITLE
Change writer to author where necessary

### PR DESCRIPTION
**Pull request by amrandahvur**

### CODE REVIEW

**Comment by amrandahvur** [comment]
@Adrianzatreanu I think i have implemented what you specified. Please Check ?

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -42,7 +42,7 @@ coala 0.8.0 - grizzly
 For this release, we have had 46 developers from around the world contributing
```
Comment (adtac) [inline RELEASE_NOTES.rst]: @amrandahvur I wouldn't change the release notes. They are of historic value and shouldn't be changed IMO.

**Comment by adtac** [comment]
unack https://github.com/coala/coala/pull/2851/commits/fbbf4773c41e38342fc7886cacf53c2d0f27f6ee

**Comment by adtac** [comment]
ack https://github.com/coala/coala/pull/2851/commits/294861092c7c8951062dfc8d6ab87dc9bf2f228b

File: RELEASE_NOTES.rst (hunk 1 above)
Comment (amrandahvur) [inline RELEASE_NOTES.rst]: @adtac I havent quite understood what you mean to say, Could you explain ? Should i revert back all "author" to "writer" in RELEASE_NOTES.rst ??

Comment (adtac) [inline RELEASE_NOTES.rst]: remove the commit that changes stuff in the release notes

Comment (amrandahvur) [inline RELEASE_NOTES.rst]: Sure, Working on it :D

Comment (adtac) [inline RELEASE_NOTES.rst]: looks good

**Comment by adtac** [comment]
ack https://github.com/coala/coala/pull/2851/commits/4da409c9a0496955145dc6b2ed76e39056feab91

**Comment by adtac** [comment]
@rultor merge

**Comment by amrandahvur** [comment]
Thank you for merging my first PR :)

**Comment by adtac** [comment]
Hey @amrandahvur , this is your first contribution, right? Congrats and welcome to the coalaians! :)

You can help us a lot by filling out http://coala.io/newform!

Rultor will now automatically release your changes as a prerelease so users can experience the goodness of your changes right now.

See also https://github.com/coala-analyzer/coala/wiki/coala-Steam-Offer btw.

Also, you can now move on to difficulty/low

**Pull request merged**